ccle_tasks/data/stjude_rms/participant.tsv

```diff
@@ -1,70 +0,0 @@
-	entity:participant_id	attr_age_at_diagnosis	attr_ethnicity	attr_race	attr_sex
-0	SJ000004	9.31	Not Hispanic or Latino	White	Male
```

Comment:
same here. otherwise we wouldn't know how to reproduce.

Reply:
resolved in new PR